ci/test/04_install.sh

```diff
@@ -92,6 +92,9 @@ elif [ "$CI_USE_APT_INSTALL" != "no" ]; then
     # TODO: drop this once we can use newer images in GCE
     CI_EXEC_ROOT add-apt-repository ppa:hadret/bpfcc
   fi
+  if [[ -n "${APPEND_APT_SOURCES_LIST}" ]]; then
+    CI_EXEC echo "${APPEND_APT_SOURCES_LIST}" >> /etc/apt/sources.list
```

Comment:
fa33242a1ab340aad177021f6f12073081615f86:
```suggestion
    CI_EXEC_ROOT echo "${APPEND_APT_SOURCES_LIST}" >> /etc/apt/sources.list
```